elements/head.html

```diff
@@ -0,0 +1,50 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        <!-- Actual Tag Name; Whatever the tag name is.  -->
+        &lt;tagname>
```

Comment:
tagname is a placeholder for whatever the tag would be, in this case head should be put there, because it's the head.html file. to be honest, i dont know what happened to the head.html file, because i did finish it at one point in time. but possible it is in a branch i have not merged to master? ill check on that.